src/doc/trpl/guessing-game.md

```diff
@@ -372,15 +371,13 @@ Now, without changing any of our code, let’s build our project:
 ```bash
 $ cargo build
     Updating registry `https://github.com/rust-lang/crates.io-index`
- Downloading rand v0.3.8
+ Downloading rand v0.3.0
  Downloading libc v0.1.6
    Compiling libc v0.1.6
-   Compiling rand v0.3.8
+   Compiling rand v0.3.0
    Compiling guessing_game v0.1.0 (file:///home/you/projects/guessing_game)
 ```
 
-(You may see different versions, of course.)
```

Comment:
This is actually important, as if a 0.3.9 comes out, it will get used. or a 0.3.10. just not 0.4.0